src/components/Profile.tsx

```diff
@@ -28,9 +28,9 @@ export default function Profile({ session, username }: Props) {
   const { name, image } = getUserSession(session);
 
   return (
-    <div className="p-4 w-full rounded-lg bg-pod-purple">
-      <div className="flex flex-col justify-center lg:justify-between lg:flex-row gap-4 items-center min-w-[270px]">
-        <div className="relative rounded-full min-h-[200px] min-w-[200px] border-white border-8">
+    <div className="p-4 w-full rounded-lg bg-pod-purple h-fit">
```

Comment:
What does this `h-fit` does?

Reply:
It's a leak from previous tests. Not needed anymore, just removed it.